### PR TITLE
Check status server

### PR DESCRIPTION
**Pull request by jewelltaylor**
# PR Type
[**Feature** | Fix | Documentation | Other ]

# Short Description

Clickup Ticket(s): [Make the server's "check status" endpoint](https://app.clickup.com/t/86884uj0w)

Add server endpoint that returns the value at key server_uuid if it exists. 

# Tests Added
- Test valid server uuid, test non existent server_uuid and handling fail exception.

### CODE REVIEW

File: florist/api/client.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def start(server_address: str, client: str, data_path: str, redis_host: str, red
         return JSONResponse({"error": str(ex)}, status_code=500)
 
 
-@app.get("/api/client/check_status")
+@app.get("/api/client/check_status/{client_uuid}")
```
Comment (jewelltaylor) [inline florist/api/client.py:78]: I did not include a path parameter in the check_status endpoint for client initially but circled back in the PR to add it. Lmk if you agree!

Comment (lotif) [reply florist/api/client.py:78]: How does this work when you're calling this endpoint? Something like this maybe?
```url
http://localhost:8001/api/client/check_status/1c52dc0f-8c3d-47af-8b01-0528d4d37bfb?redis_host=localhost&redis_port=6380
```

**Review by lotif — APPROVED** [review]
Looks good, just one question.

File: florist/api/client.py (hunk 1 above)
Comment (jewelltaylor) [reply florist/api/client.py:78]: Yup! Here is an example: `http://localhost:8001/api/client/check_status/fe764812-94a1-4c18-bb30-656f6ef67968?redis_host=localhost&redis_port=6380`

**Pull request merged**